apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -64,6 +64,17 @@ function getSizeInfo({ isVertical, checkboxes, dense, height }) {
   };
 }
 
+function SearchMessage(width) {
+  const styleSearchMsg = {
+    fontSize: '16px',
+    padding: '16px 0',
+    textAlign: 'center',
+    minWidth: width,
+  };
+
+  return <div style={styleSearchMsg}>There are no matches for your search.</div>;
```

Comment:
Wrap the string in `{translator.get('There are no matches for your search.')}` like in ListboxInline.jsx, and maybe you can  reuse the translation string from the old listbox?

Reply:
I'll do this way, looks like the translation from the old listbox ('Listbox.NoMatchesForYourTerms') will not work